### DISCUSSION TITLE
Screenshots gallery for readme.md

### DISCUSSION THREAD

**Discussion by Vladimir-Ciuculescu** [community/community#64531] [Other Feature Feedback, Questions, & Ideas]
Screenshots gallery

Does anyone know any online app or tool that can help you generate such image with screenshots ?

![authflow](https://github.com/community/community/assets/97725392/556a6137-f1eb-4a04-9b4c-4fb766887d72)

**Comment by arafajs** [comment]
- Canva.
- Search for "Mobile Template" and customize it as you want.
- If you want "App template" you can visit Behance.
